metadata/_category/_template.md

```diff
@@ -0,0 +1,69 @@
+<!-- The filename should match the OpenSGID layer_name. For example, roads.md. The category will be inferred from the directory structure -->
+
+# Title
+
+<!-- The human-readable layer name that should match the pageTitle item in the data page's metadata. If it's a statewide layer, it should include "Utah", e.g. "Utah Roads" -->
+
+## Brief Summary
+
+<!-- A super-short, one-liner description of the dataset. Corresponds to the pageDescription item in the data page's metadata -->
+
+## Summary
+
+<!-- A brief (<2048 characters for AGOL) explanation of the dataset to give the user a high-level overview of what the layer is when skimming lists of datasets. Will be the first description people see in Hub open data and should match the Summary section of the layer's data page  -->
+
+## Description
+
+<!-- A more in-depth explanation of the dataset, where it came from and how its created, so the user can decide if its what they need. -->
+
+### What is the dataset
+
+### What is the purpose of the dataset
+
+### What does the dataset represent
+
+### How was the dataset created
+
+### How reliable and accurate is the dataset
```

Comment:
do we want the input here to be open-ended or limit the input to standardized wording? I kind of like the idea of standardized verbiage for this section.

Reply:
Do you have any ideas for standardized reliability/accuracy ratings? Maybe something out of the ranking efforts?

Reply:
I don't have any solid ideas yet. Maybe we go back to something similar to our gold, silver, and bronze ratings as a base and then elaborate if/when needed.